pkg/api/defaults-kubelet.go

```diff
@@ -78,6 +78,7 @@ func (cs *ContainerService) setKubeletConfig() {
 		"--pod-max-pids":                    strconv.Itoa(DefaultKubeletPodMaxPIDs),
 		"--image-pull-progress-deadline":    "30m",
 		"--enforce-node-allocatable":        "pods",
+		"--read-only-port":                  "0",
```

Comment:
Please add the flag to unit tests (after applying Jack's comment about conditionally adding this flag)